tests/streamingcli/resources/jupyter/with_init/notebook4.ipynb

```diff
@@ -10,20 +10,6 @@
     "%reload_ext streamingcli.jupyter.integrations"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "%flink_execute_sql_file --path ./init.sql"
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%%\n"
-    }
-   }
-  },
```

Comment:
Why this is not needed anymore?

Reply:
Now scli executes `init.sql` automatically so we don't need to force it with a command.